src/components/core/RightCol.vue

```diff
@@ -18,11 +18,18 @@
                             showCreditRefreshNotification = false"></button>
           </header>
           <section class="modal-card-body custom-card-transactions">
-            <div class="modal-container" ref="transactionsContainer" @scroll="handleScroll"> 
+            <div class="modal-container custom-modal-container"  ref="transactionsContainer" @scroll="handleScroll"> 
               <div
                 class=" custom-card is-flex-direction-column is-align-items-center is-justify-content-space-between mb-4"
               >
                 <AllTrs />
+                <div v-if="isLoadingTransactionsBatch" class="transactions-loader-container">
```

Comment:
As you did it, we hit the bottom of the page and only then does the loading indicator shows up, but *below* the bottom of the page. So we need to scroll a bit more to see it.

If the `.transactions-loader-container` were to be always visible, then there would be this 40px gap at the end of the list that would be shown before the loading would occur, and so we would see the loading-indicator.

Reply:
i didn't want to show an empty space at the bottom of the div, but you are right i guess it is more important to make the loader more visible for the user. thanks

Reply:
I have a suggestion which will make both of us satistied: put the loading-indicator in the modal footer. That way it will be clearly visible.

Reply:
i agree that the loading-indicator should be always visible, so i moved it to the footer as you said, let me know if the result is satisfying.